public/app/features/alerting/unified/mocks/server/handlers/prometheus.ts

```diff
@@ -0,0 +1,12 @@
+import { http, HttpResponse } from 'msw';
+
+export const getPrometheusRulesHandler = () => {
+  return http.get(`/api/prometheus/:datasourceUid/api/v1/rules`, () => {
+    return HttpResponse.json({
+      data: { groups: [] },
```

Comment:
Recording rule heuristics check if we can fetch rules for a datasource, to work out if we can use it or not. It only checks for a successful response (which also requires it to have `groups` property), so this is the bare minimum to make `loki` show up in my recording rule editor tests

(TODO: Make the response more sensible)